docs/ai/toc.yml

```diff
@@ -4,6 +4,8 @@ items:
 - name: Concepts
   items: []
 - name: How-to articles
-  items: []
+  items:
+  - name: Use custom and local AI models with the Semantic Kernel SDK
```

Comment:
```suggestion
  - name: Use AI models with the Semantic Kernel SDK
```

Reply:
I would keep "custom and local" because this doc doesn't cover how to use the "standard" models (OpenAI, AOAI). So the important distinction with this article is that it explains how to integrate something custom that isn't part of SK out of the box